tests/spread/smoketests/metafiles/actions/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: create a charm that use actions in charmcraft.yaml
+
+include: 
+  - tests/
+
+prepare: |
+  tests.pkgs install unzip
+  charmcraft init --project-dir=charm
+  cat <<- EOF >> charmcraft.yaml
+  actions:
+    validate-configuration:
```

Comment:
Here's an example of where validation for `actions` could come in handy. I'm guessing eventually we're going to want to ensure that the charm has an `_on_validate_configuration_action` method based on this? (That feels like something we should be catching before it becomes Somebody Else's Problem™.)